src/main/groovy/info/solidsoft/gradle/pitest/PitestTask.groovy

```diff
@@ -362,35 +362,34 @@ class PitestTask extends JavaExec {
     private Map<String, String> prepareMapWithClasspathConfiguration() {
         if (useAdditionalClasspathFile.get()) {
             fillAdditionalClasspathFileWithClasspathElements()
-            return [classPathFile: getAdditionalClasspathFile().asFile.get().absolutePath]
-        } else {
-            return [classPath: getAdditionalClasspath().files.join(',')]
+            return [classPathFile:getAdditionalClasspathFile().asFile.get().absolutePath]
         }
+        return [classPath:getAdditionalClasspath().files.join(',')]
     }
 
     private void fillAdditionalClasspathFileWithClasspathElements() {
-        String classpathElementsAsFileContent = getAdditionalClasspath().files.collect { it.getAbsolutePath() }.join(System.lineSeparator())
+        String classpathElementsAsFileContent = getAdditionalClasspath().files*.getAbsolutePath().join(System.lineSeparator())
         //"withWriter" as "file << content" works in append mode (instead of overwrite one)
-        getAdditionalClasspathFile().asFile.get().withWriter() {
-            it << classpathElementsAsFileContent
+        getAdditionalClasspathFile().asFile.get().withWriter { writer ->
+            writer << classpathElementsAsFileContent
         }
     }
 
+    @SuppressWarnings("IfStatementCouldBeTernary")
     private Map<String, String> prepareMapWithIncrementalAnalysisConfiguration() {
         if (enableDefaultIncrementalAnalysis.getOrNull()) {
-            return [historyInputLocation : getHistoryInputLocation()?.getOrNull()?.asFile?.absolutePath ?: getDefaultFileForHistoryData().asFile.get().absolutePath,
-                    historyOutputLocation: getHistoryOutputLocation()?.getOrNull()?.asFile?.absolutePath ?: getDefaultFileForHistoryData().asFile.get().absolutePath]
-        } else {
-            return [historyInputLocation: getHistoryInputLocation()?.getOrNull()?.asFile?.absolutePath,
-                    historyOutputLocation: getHistoryOutputLocation()?.getOrNull()?.asFile?.absolutePath]
+            return [historyInputLocation:getHistoryInputLocation()?.getOrNull()?.asFile?.absolutePath ?: getDefaultFileForHistoryData().asFile.get().absolutePath,
+                    historyOutputLocation:getHistoryOutputLocation()?.getOrNull()?.asFile?.absolutePath ?: getDefaultFileForHistoryData().asFile.get().absolutePath,]
         }
+        return [historyInputLocation:getHistoryInputLocation()?.getOrNull()?.asFile?.absolutePath,
+                historyOutputLocation:getHistoryOutputLocation()?.getOrNull()?.asFile?.absolutePath,]
```

Comment:
It's redundant, but the variant with `else` looks more natural for me in that situation. Could we bring it back?